packages/hash/frontend/src/components/layout/PageHeader/ActionsDropdown.tsx

```diff
@@ -56,16 +56,23 @@ export const ActionsDropdown: React.FC<{
       <IconButton
         size="medium"
         rounded
-        sx={{
+        sx={({ palette }) => ({
           mr: {
             xs: 1,
             md: 1.5,
           },
-          color: open ? theme.palette.common.white : theme.palette.gray[40],
-          backgroundColor: open
-            ? theme.palette.blue["70"]
-            : theme.palette.gray[20],
-        }}
+          color: open ? palette.common.white : palette.gray[40],
+          backgroundColor: open ? palette.blue["70"] : palette.gray[20],
+
+          ":hover": {
+            boxShadow: "0px 0px 0px 2px #FFFFFF, 0px 0px 0px 5px #C1CFDE",
```

Comment:
**Blocking:** we should always use the MUI palette to access colours when possible.
```suggestion
            boxShadow: `0px 0px 0px 2px ${palette.white}, 0px 0px 0px 5px ${palette.gray[40]}`,
```

Reply:
Couple of things I'd like to add
1) Using box-shadow to represent the focus state is pretty cool and seems easier to implement compared to what we are currently doing for our buttons. Will adopt this approach, thanks!
2) I think it should be `transparent` instead of white, since that is meant to represent the space between the IconButton and the focus ring and doesn't necessarily have to be white.

Reply:
@teenoh tried out point 2., but applying `transparent` instead of `white` results in the complete box-shadow taking up the color applied at the end.

Reply:
Ahh I see. I guess in this case, we can leave as is, but for scenarios where the component can be laid on multiple backgrounds, we'd have to use the approach we use for our [buttons](https://github.com/hashintel/hash/blob/87b8361d16613042722054c42c94a58b8862ea1b/packages/hash/frontend/src/theme/components/inputs/MuiButtonThemeOptions.ts#L67-L79), which is using the border of the `:before` or `:after` pseudo-element